docs/en_US/Overview.md

```diff
@@ -46,6 +46,38 @@ For each experiment, user only needs to define a search space and update a few l
 
 More details about how to run an experiment, please refer to [Get Started](Tutorial/QuickStart.md).
 
+## Core Capacities
+
+Hyperparameter tuning is a core capacity of NNI, it can be used to tune hyperparameters in machine learning and deep learning models, parameters in complex systems, etc. While hyperparameter tuning is only one step in designing a machine learning or deep learning model. Thus, NNI is evolved to help more on finding a better model with the following (pipelined) components.
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/16907603/70137129-3f549700-16c8-11ea-8204-3202b90f24a9.png" alt="drawing" width="700"/>
```

Comment:
The graph is confusing to me,
1. It looks the whole process can apply to all kinds of work, but actually not. For example, feature selections mostly applies to traditional machie learning algorithm, but NAS, model compression applies to deep learning.
2. what does the different colors mean? And is it important to highlight with different colors?

Reply:
good point, will update this figure and description